internal/security/spiffetokenprovider/init.go

```diff
@@ -164,7 +164,7 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 	//
 
 	// Handle gettoken endpoint
-	http.HandleFunc("/api/v2/gettoken", func(w http.ResponseWriter, r *http.Request) {
+	http.HandleFunc("/api/v3/gettoken", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
This should be using the API  constants from core contracts
```suggestion
	http.HandleFunc(common.ApiBase +"/gettoken", func(w http.ResponseWriter, r *http.Request) {
```

Reply:
updated